demo/patternNBandImport.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 S. Pawar, S. Semper
+#     https://www.tu-ilmenau.de/it-ems/
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Demo to import of narrowband pattern data stored as .mat file to produce an EADF object
+------------------------------------
+
+>>> cd demo
+>>> python patternImport.py
+"""
+
+import eadf
+import numpy as np
+import matplotlib.pyplot as plt
+
+if __name__ == "__main__":
+
+    # directory location and name of pattern .mat file
+    patternDir = "TUI_SPUCA_2x8_I_20MHz_pattern.mat"
```

Comment:
variable name is kinda misleading, since it is not a directory, but a file

Reply:
if the file isn't in the same folder as the demo file, we would need to add the whole directory path, right? That isn't the case - so we can replace it with patternFile?

Reply:
you could name it "patternPath". I think this should be clear then